pepys_admin/admin_cli.py

```diff
@@ -148,6 +150,40 @@ def do_export(self, arg):
             with self.datastore.session_scope():
                 self.datastore.export_datafile(selected_datafile_id, datafilename)
 
+    def do_export_all(self, arg):
+        "Start the export all datafiles process"
+        export_flag = input("Do you want to export all Datafiles. (Y/n)\n")
+        if export_flag in ["", "Y", "y"]:
+            while True:
+                folder_name = input(
+                    "Please provide folder name (Press Enter for auto generated folder):"
+                )
+                if folder_name:
+                    if os.path.isdir(folder_name):
+                        print("{} already exists.\n".format(folder_name))
+                    else:
+                        os.mkdir(folder_name)
+                        break
+                else:
+                    folder_name = datetime.datetime.now().strftime(

```

Comment:
we can't have ":" in pathnames for all operating systems. Happy with 8-digit date then 6-digit time `20200320_1855`

Reply:
Fixed.